src/pool/pool_disjoint.c

```diff
@@ -570,6 +572,13 @@ static void *disjoint_pool_allocate(disjoint_pool_t *pool, size_t size) {
     return ptr;
 }
 
+static void free_slab(void *unused, void *slab) {
```

Comment:
please comment when this function would be called

Reply:
Done